homeassistant/components/tuya/light.py

```diff
@@ -350,6 +350,11 @@ class TuyaLightEntityDescription(LightEntityDescription):
             brightness=DPCode.BRIGHT_VALUE,
             color_temp=DPCode.TEMP_VALUE,
         ),
+        TuyaLightEntityDescription(
+            key=DPCode.SWITCH_LED,
```

Comment:
This is not part of the specification that is listed in the link above these descriptions.

The least we should do, is add a comment where this is coming from.